src/BSON/Binary.stub.php

```diff
@@ -63,13 +63,13 @@ final class Binary implements BinaryInterface, \JsonSerializable, Type, \Seriali
      */
     public const TYPE_USER_DEFINED = UNKNOWN;
 
-    public final function __construct(string $data, int $type) {}
+    final public function __construct(string $data, int $type) {}
```

Comment:
This PR should only change `__set_state` methods. Please leave these other corrections to [PHPC-2136](https://jira.mongodb.org/browse/PHPC-2136) (as mentioned in https://github.com/mongodb/mongo-php-driver/pull/1357#discussion_r967541061) so we don't end up with unrelated changes in the final commit.

Reply:
Done, please review again

Reply:
There is one CI fail I will fix it and request review again

Reply:
Done